### DISCUSSION TITLE
How can I disable validation when onBlur mode

### DISCUSSION THREAD

**Discussion by wai15985** [react-hook-form/react-hook-form#7813] [General] [answered]
Codesandbox
https://codesandbox.io/s/pedantic-bush-t8xe2?file=/src/App.js:0-585

Issue:
I am tring to create a show password function, Is it possible disable validation when i click on show button?

**Comment by bluebill1049** [comment, marked as answer]
for custom behaviour, perhaps you want to set mode to `onSubmit` and use `trigger`.

**Reply by wai15985** [reply]
I also use isValid to toggle submit button, how can I validation isValid using custom onBlur event?

**Reply by bluebill1049** [reply]
It will get updated if you have subscribed to `isValid`.

**Reply by wai15985** [reply]
I see. I thought it is only applicable with onChange, onTouched, and onBlur mode. Thanks for help
